DatadogSessionReplay/Sources/Recorder/Utilities/CGPath+SessionReplay.swift

```diff
@@ -0,0 +1,69 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+#if os(iOS)
+
+import CoreGraphics
+import DatadogInternal
+import Foundation
+
+#if $RetroactiveAttribute
+extension CGPath: @retroactive DatadogExtended {}
+#else
+extension CGPath: DatadogExtended {}
+#endif
```

Comment:
Silence warnings for retroactive conformances while keeping Swift 5.9 compatibility. There are a few of these warnings in other modules that we can fix using this technique.